cime_config/machines/config_machines.xml

```diff
@@ -2937,7 +2937,7 @@
           <command name="use">/soft/restricted/CNDA/updates/modulefiles</command>
         </modules>
         <modules compiler="!gnu">
-          <command name="load">oneapi/eng-compiler/2022.12.30.003</command>
+          <command name="load">oneapi/eng-compiler/2023.05.15.007</command>
           <command name="load">mpich/52.2/icc-all-pmix-gpu</command>
```

Comment:
@amametjanov Can you also consider removing/commenting the mpich version since a new, compatible version (`mpich/52.2-256/icc-all-pmix-gpu`) of MPICH gets loaded with 2023.05.15.007